card/db/schema.rb

```diff
@@ -38,7 +38,7 @@
   create_table "card_changes", force: :cascade do |t|
     t.integer "card_action_id", limit: 4
     t.integer "field",          limit: 4
-    t.text    "value",          limit: 65535
+    t.text    "value",          limit: 16777215
```

Comment:
But do we know how/why or is this maybe accidental?

I'm guessing there are some details of all of it that are unclear at this point. I'm just curious what we do and don't know.